_config.yml

```diff
@@ -37,14 +37,14 @@ cache:
 # Site Information Settings
 # ---------------------------------------------------------------
 
-# To get or check favicons visit: https://realfavicongenerator.net
-# Put your favicons into `hexo-site/source/` (recommend) or `hexo-site/themes/next/source/images/` directory.
+# Default NexT favicons are placed in `hexo-site/themes/next/source/images/` directory.
+# You can replace them with your own favicons.
 
-# Default NexT favicons placed in `hexo-site/themes/next/source/images/` directory.
-# And if you want to place your icons in `hexo-site/source/` root directory, you must remove `/images` prefix from pathes.
+# For example, you can put your favicons in `hexo-site/source/images/` directory.
+# Then need to rename them and change the settings below, otherwise icons from Next will rewrite your custom icons in Hexo.
+# And if you want to place your icons in `hexo-site/source/` root directory, you must remove `/images` prefix from paths.
 
-# For example, you put your favicons into `hexo-site/source/images` directory.
-# Then need to rename & redefine them on any other names, otherwise icons from Next will rewrite your custom icons in Hexo.
+# You can generate custom favicons on this site: https://realfavicongenerator.net
```

Comment:
I think favicon description need to cutted out to docs site with just defined permalink here. Same with other items (with footer below, for example).

But for now I see PR title called «Refactoring `font` settings in `_config.yml`», so, I don't understand why there are excess changes exists.

Reply:
@ivan-nginx Now?

Reply:
Just add: https://theme-next.org/docs/getting-started/index.html#Configuring-Favicon

Similar to other sections.

Reply:
Sure, I'll check the docs.
Besides, when shall we implement this: https://github.com/theme-next/hexo-theme-next/issues/703 ?

Reply:
We will come to this soon, for now it's not high priority.

HP for now:

1. Move all excess comments from `_config.yml` to documentation ordering website (as I gave example above).
2. Create separated docs with old settings for version 6.x (v6.theme-next.org).
3. Improve generation speed (cache still not fixed).